src/wallet/mod.rs

```diff
@@ -982,6 +982,16 @@ where
         Ok((psbt, finished))
     }
 
+    /// Override the signing options for all the software signers
+    ///
+    /// Since the signers are wrapped in `Arc` they will only be updated if it's possible to take a
```

Comment:
maybe better to fail if it's not possible to update then?
I see `Arc::get_mut` allows to check if there are shared references